<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="1">
     <widget class="QPushButton" name="submit_btn">
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="exit_btn">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QGroupBox" name="form_group">
      <property name="title">
       <string>Your information</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="name_label">
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Full Name : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="name_edit">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Participate : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="absolutly_check">
           <property name="text">
            <string>Absolutly</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="maybe_check">
           <property name="text">
            <string>Maybe</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="sorry_check">
           <property name="text">
            <string>Sorry, I'll just pay</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="vegetarian_label">
           <property name="text">
            <string>Vegtarian : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="yes_radio">
           <property name="text">
            <string>yes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="no_radio">
           <property name="text">
            <string>No</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="transportation_label">
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Transortation : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="transportation_combo">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="mambers_label">
           <property name="text">
            <string>Members : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="members_spin">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Anything you want to say to us</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="input_text"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QLabel" name="sub_title">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Join us and celebrate together!</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="title">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>48</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>48</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Serif</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ngan Dang Ngan Kong DekBen &amp; ......</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
